src/components/mobile/ExploreMobile.tsx

```diff
@@ -90,23 +90,21 @@ export default function ExploreMobile() {
               }}
               />
           {/* Icons */}
-          <div className="icon-container"
-            style={{ top: 10, right: 15 }}>
-            <FontAwesomeIcon
-              icon={faFilter}
-              onClick={() => handleFilterToggle(!showMobileFilters, false)}
-              className={'icon'}
-              color={'#455a64'}
-              size={"2x"} />
+          <div className="icon-container" style={{ top: 10, left: 10 }}>
```

Comment:
Don't think it's great practice to be relying on inline styles

Reply:
fixed